handlers/proxy_status.go

```diff
@@ -4,13 +4,22 @@ import (
 	"net/http"
 
 	"github.com/gorilla/mux"
+	"k8s.io/client-go/tools/clientcmd/api"
+
+	kiali_business "github.com/kiali/kiali/business"
+	"github.com/kiali/kiali/kubernetes"
 )
 
 func ConfigDump(w http.ResponseWriter, r *http.Request) {
 	params := mux.Vars(r)
 
 	// Get business layer
-	business, err := getBusiness(r)
+	kialiToken, err := kubernetes.GetKialiToken()
```

Comment:
I'm just documenting this here for me (and this is why I want to get my PRs merged soon)... this `GetKialiToken() `method will be renamed in the future to `GetKialiTokenForHomeCluster()`.